dag/main.yml

```diff
@@ -603,12 +603,12 @@ steps:
     - data://garden/survey/2023-08-04/trust_surveys
 
   # World Bank GenderStats
-  data://meadow/wb/2023-12-29/gender_statistics:
-    - snapshot://wb/2023-12-29/gender_statistics.feather
-  data://garden/wb/2023-12-29/gender_statistics:
-    - data://meadow/wb/2023-12-29/gender_statistics
-  data://grapher/wb/2023-12-29/gender_statistics:
-    - data://garden/wb/2023-12-29/gender_statistics
+  data://meadow/wb/2024-06-10/gender_statistics:
+    - snapshot://wb/2024-06-10/gender_statistics.feather
+  data://garden/wb/2024-06-10/gender_statistics:
+    - data://meadow/wb/2024-06-10/gender_statistics
+  data://grapher/wb/2024-06-10/gender_statistics:
+    - data://garden/wb/2024-06-10/gender_statistics
 
   # Missing Data - Children out of school
   data://garden/missing_data/2024-03-26/children_out_of_school:
```

Comment:
The ETL is failing because there is this dataset that is using the previous version of the Gender dataset. Change it to the current one and check if there are no issues with the change.

Reply:
ahh thanks a lot for picking this up! 🙇‍♀️